addons/account/models/account_bank_statement.py

```diff
@@ -74,7 +74,8 @@ class AccountBankStatement(models.Model):
     @api.one
     @api.depends('line_ids', 'balance_start', 'line_ids.amount', 'balance_end_real')
     def _end_balance(self):
-        self.total_entry_encoding = sum([line.amount for line in self.line_ids])
+        total = sum([line.amount for line in self.line_ids])
```

Comment:
Remove the brackets to avoid double-looping.

```suggestion
        total = sum(line.amount for line in self.line_ids)
```